integration/base/github/metadata.yaml

```diff
@@ -2,4 +2,5 @@ customer_id: "-Am-_6i5pw0u4AbspOwKN4lZUCn49u_G"
 installation_id: "RULNveQLrLj4GDum1yQhvKAcABh0GcLY"
 release_version: "0.0.4"
 set_github_contents: "replicatedhq/test-charts:/release.yml:master:.ship"
-disable_online: true
\ No newline at end of file
+disable_online: true
+skip_cleanup: false
```

Comment:
intentional?

Reply:
I added `skip_cleanup: false` to most of the tests that didn't have it yet